core/migrations/0001_initial.py

```diff
@@ -0,0 +1,156 @@
+# Generated by Django 5.0.4 on 2024-05-14 18:46
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [  # noqa: RUF012
+        ("contenttypes", "0002_remove_content_type_name"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [  # noqa: RUF012

```

Comment:
MIgrations are excluded from ruff tests: https://github.com/pymasterspl/reddit/blob/494c9c9a743d0090c3eff5ebd50df9af91a2f682/pyproject.toml#L24
please remove those #noqa notes and if this still fails, we need to fix ruff config.